src/js/app.js

```diff
@@ -1134,7 +1140,7 @@ function addMessage(msg, chatId) {
     var profile = chats[chatId].participantProfiles[msg.info.from];
     var name = profile && profile.name;
     if (name) {
-      var nameEl = $('<small>').text(name).css({color: profile.color, 'margin-bottom':2,display:'block','font-weight':'bold'});
+      var nameEl = $('<small onclick="addMention(this)">').text(name).css({color: profile.color, 'margin-bottom':2,display:'block','font-weight':'bold'}).addClass('msgSenderName');
```

Comment:
```suggestion
      var nameEl = $('<small>').click(() => addMention(this)).text(name).css({color: profile.color}).addClass('msgSenderName');
```
click() handler & move style to .css

Reply:
`@${pub.slice(1,12)}` would be a more advanced mention markup that works with changing names. But we can start simple.